.github/workflows/test-and-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Test and Lint
```

Comment:
How is this different from the build template?

Reply:
It is really the same template but this one runs against all branches, on push to branch or on PR, and not on a cron.

Build can basically be thought of as "Is master successful"

Reply:
Got it, thanks. Maybe it should be `Build - master` to make that more explicit?